x/emissions/keeper/msgserver/msg_server_topics.go

```diff
@@ -58,6 +58,7 @@ func (ms msgServer) CreateNewTopic(ctx context.Context, msg *types.MsgCreateNewT
 		PNorm:           msg.PNorm,
 		AlphaRegret:     msg.AlphaRegret,
 		AllowNegative:   msg.AllowNegative,
+		Epsilon:         msg.Epsilon,
```

Comment:
so we just let the topic creator set an epsilon to whatever they want with no validation at all?


at minimum shouldn't we prevent setting to zero or negative numbers, epsilon is supposed to exist to prevent division by zero

Reply:
There is preventing setting it to zero or negative, see `x/emissions/types/msg_create_topic.go`

Reply:
very weird that lives in a separate file than the rest of the msg server... ok